examples/basic/android/settings.gradle

```diff
@@ -8,5 +8,9 @@ include ':app',
 //project(':react-native-video').projectDir = new File(rootProject.projectDir, '../../android')
```

Comment:
Is this commented out code still needed?

Reply:
Removed.